client/src/components/profile/components/Camper.tsx

```diff
@@ -129,7 +129,7 @@ function Camper({
           <br />
           <p className='text-center yearsTopContributor'>
             <FontAwesomeIcon icon={faAward} />{' '}
-            <Link to={'/top-contributors'}>{t('profile.contributor')}</Link>
+            <Link to={t('links:top-contributors')}>{t('profile.contributor')}</Link>
```

Comment:
```suggestion
            <Link to={t('links:top-contributors')}>
              {t('profile.contributor')}
            </Link>
```
My fault. Forgot about the linter.

Reply:
Can You please check now, I have done what you had directed me to do.